tdrs-backend/tdpservice/parsers/schema_defs/ssp/m5.py

```diff
@@ -0,0 +1,367 @@
+                validators.if_then_validator(
+                    condition_field="DATE_OF_BIRTH",
+                    condition_function=validators.olderThan(18),
+                    result_field="REC_OASDI_INSURANCE",
+                    result_function=validators.isInLimits(1, 2),
+                ),
+                validators.if_then_validator(
+                    condition_field="FAMILY_AFFILIATION",
+                    condition_function=validators.matches(1),
+                    result_field="REC_FEDERAL_DISABILITY",
+                    result_function=validators.isInLimits(1, 2),
+                ),
+            ],
+            fields=[
+                Field(
+                    item="0",
+                    name="RecordType",
+                    type="string",
+                    startIndex=0,
+                    endIndex=2,
+                    required=True,
+                    validators=[],
+                ),
+                Field(
+                    item="3",
+                    name="RPT_MONTH_YEAR",
+                    type="number",
+                    startIndex=2,
+                    endIndex=8,
+                    required=True,
+                    validators=[
+                        validators.dateYearIsLargerThan(1998),
+                        validators.dateMonthIsValid(),
+                    ],
+                ),
+                Field(
+                    item="5",
+                    name="CASE_NUMBER",
+                    type="string",
+                    startIndex=8,
+                    endIndex=19,
+                    required=True,
+                    validators=[validators.isAlphaNumeric()],
+                ),
+                Field(
+                    item="13",
+                    name="FAMILY_AFFILIATION",
+                    type="number",
+                    startIndex=19,
+                    endIndex=20,
+                    required=True,
+                    validators=[validators.isInLimits(1, 5)],
+                ),
+                Field(
+                    item="14",
+                    name="DATE_OF_BIRTH",
+                    type="string",
+                    startIndex=20,
+                    endIndex=28,
+                    required=True,
+                    validators=[
+                        validators.dateYearIsLargerThan(1900),
+                        validators.dateMonthIsValid(),
+                    ],
+                ),
+                TransformField(
+                    transform_func=ssp_ssn_decryption_func,
+                    item="15",
+                    name="SSN",
+                    type="string",
+                    startIndex=28,
+                    endIndex=37,
+                    required=True,
+                    validators=[validators.validateSSN()],
+                    is_encrypted=False,
+                ),
+                Field(
+                    item="16A",
+                    name="RACE_HISPANIC",
```

Comment:
@elipe17 @jtimpe are space-fill accounted for here yet? same q for all race fields and all other fields where space is listed in workbook as valid value.

Reply:
i think i recall that space-fill is accounted for with required = False. is this correct?

Reply:
m4 looks to account for space fill in the race fields. you are correct that `required=False` means space-fill is allowed